modules/php/Models/Player.php

```diff
@@ -122,9 +124,9 @@ public function countHand()
   {
     return Cards::countHand($this->id);
   }
-  public function setHp($hp)
+  public function getGeneralStorePref()
```

Comment:
Maybe the function could be called isAutoPickGeneralStore or something like that to make it clear it return a boolean ?